package/src/index.ts

```diff
@@ -3,3 +3,4 @@ export { addVitePlugin } from "./utils/add-vite-plugin.js";
 export { createResolver } from "./utils/create-resolver.js";
 export { defineIntegration } from "./utils/define-integration.js";
 export { watchIntegration } from "./utils/watch-integration.js";
+export { hasIntegration } from "./utils/has-integration.js";
```

Comment:
Please run `pnpm lint:fix` to order imports properly 😉

Reply:
😱 I did! I think the reordering linting is broked